data/ArchLinux.yaml

```diff
@@ -0,0 +1,4 @@
+---
+nginx::daemon_user: http
```

Comment:
Archlinux \o/

I'm pretty sure filename should be Archlinux.yaml

Reply:
I copied it from ```params.pp``` where it's also ```ArchLinux```. Might be good to verify there's tests for it.